content/rest/reference/issues.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Issues
+intro: 'The Issues API enables the authenticated user to list and manage the issues created on different levels of repositories.'
```

Comment:
I would suggest making this a bit more generic since this API is so broad:
```suggestion
intro: 'The Issues API enables you to view and manage issues, including issue assignees, comments, labels, and milestones.'
```